tensorflow/compiler/xla/service/gpu/tests/gemm_rewrite_test.cc

```diff
@@ -136,17 +136,16 @@ ENTRY AddDotsFunc {
     StatusOr<bool> filecheck_result_cublas =
         RunFileCheck(optimized_module->ToString(),
                      R"(
-; CHECK:    \"selected_algorithm\":\"-1\"
+; CHECK:    custom_call_target="__cublas$gemm"
       )");
     TF_ASSERT_OK(filecheck_result_cublas.status());
     EXPECT_TRUE(filecheck_result_cublas.ValueOrDie());
     EXPECT_TRUE(RunAndCompare(*get_module(), ErrorSpec{1e-5, 1e-5}));
   } else {
-    // With cublaslt enabled, selected_algorithm is se::blas::kNoAlgorithm
     StatusOr<bool> filecheck_result_cublas =
         RunFileCheck(optimized_module->ToString(),
                      R"(
-; CHECK:    \"selected_algorithm\":\"-4\"
+; CHECK:    custom_call_target="__cublas$gemm"
```

Comment:
This is the cublasLt case, right? Shouldnt this check for custom call "__cublas$lt$matmul"?

Reply:
Not until cublasLt is on by default, which it isn't yet.  I'm reviewing that PR internally at Google.

Reply:
But this test is doing a direct check on the debug flag for cublaslt. Whether cublaslt is enabled by default or not doesn't matter. It only matters if this flag is set.

Reply:
Indeed you are right :)

Reply:
In principle, yes. The problem is that when cuBLASLt is enabled externally by setting `XLA_FLAGS=--xla_gpu_enable_cublaslt=true` in the test environment, all other tests based on `GemmRewriteTest` fail as well because they attempt to match the same pattern. Considering that the test itself switches cuBLASLt on for unit tests that are based on `CublasLtMatmulRewriteTest`, possible solutions could be to split `SimpleRewriteDeterministic` into two tests, or to remove the part that assumes cuBLASLt to be active.